datastore/cloud-client/snippets_test.py

```diff
@@ -14,26 +14,50 @@
 import os
 
 import backoff
+
+from google.api_core import exceptions
+from google.api_core.retry import Retry
+
+
 from google.cloud import datastore
+
 import pytest
 
 import snippets
 
+
 PROJECT = os.environ["GOOGLE_CLOUD_PROJECT"]
 
 
+_RETRIABLE_TYPES = [
```

Comment:
There have been discussions about using the default retry behavior, which only includes the transient errors listed here:

https://github.com/googleapis/python-api-core/blob/main/google/api_core/retry.py#L100

It looks like they cover your list plus a couple more - would this work?

Reply:
Sure, thanks for sharing the link. Is there an example of how to use this in the client library?

Reply:
I am not sure in client libraries - maybe?. You will see several instances of using it in `python-docs-samples`.

In this case, if you want to address my comment, I *think* you can just use the default behavior `retry_policy=Retry()` instead of `retry_policy = Retry(predicate=is_retryable)` and the code that goes along with it.

Reply:
Ok, changed to use the default `Retry()`